src/main/resources/migration/changelog-v5.3.0.xml

```diff
@@ -2470,4 +2470,24 @@
                 ON "COMPONENT" USING GIN ("DIRECT_DEPENDENCIES" GIN_TRGM_OPS);
         </sql>
     </changeSet>
+
+    <changeSet id="v5.3.0-15" author="sahibamittal">
+        <createTable tableName="PROJECT_METADATA">
+            <column autoIncrement="true" name="ID" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="PROJECT_METADATA_PK"/>
+            </column>
+            <column name="PROJECT_ID" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
```

Comment:
Overlooked it the first time, but we'll also need a foreign key constraint on this column.

FK constraints can be defined like so:

https://github.com/DependencyTrack/hyades-apiserver/blob/9498b4ebcbcb244e778f8b1b4f17fd9f3e774d77/src/main/resources/migration/changelog-v5.3.0.xml#L2291-L2294